daemon/analytics.c

```diff
@@ -569,6 +571,14 @@ void set_late_global_environment()
     analytics_set_data_str(&analytics_data.netdata_config_memory_mode, (char *)rrd_memory_mode_name(default_rrd_memory_mode));
     analytics_set_data(&analytics_data.netdata_config_exporting_enabled, appconfig_get_boolean(&exporting_config, CONFIG_SECTION_EXPORTING, "enabled", 1) ? "true" : "false");
 
+#ifdef DISABLE_CLOUD
+    analytics_set_data(&analytics_data.netdata_host_cloud_enabled, "false");
+#else
+    analytics_set_data(
+        &analytics_data.netdata_host_cloud_enabled,
+        appconfig_get_boolean(&cloud_config, CONFIG_SECTION_GLOBAL, "enabled", 1) ? "true" : "false");
```

Comment:
I think you need to use 

https://github.com/netdata/netdata/blob/1b819afdbba067ae6abbeac20fb8805cd1e4e3c4/libnetdata/config/appconfig.h#L152-L153

instead of 1.

I see it is the pattern, for instance

https://github.com/netdata/netdata/blob/ac3f9b171754014126edf6235d7d751f9aaa4ea5/collectors/ebpf.plugin/ebpf.c#L830-L835

Reply:
Correct, I will, thanks (I will use `CONFIG_BOOLEAN_YES`).

Btw, I also use it here, and I think it's wrong:

`analytics_set_data(&analytics_data.netdata_config_exporting_enabled, appconfig_get_boolean(&exporting_config, CONFIG_SECTION_EXPORTING, "enabled", 1) ? "true" : "false");`

I'm not sure if the default value of exporting is enabled or not (I think it's not, so if it is not found in the config, it should be disabled?) I.e. the above:

`analytics_set_data(&analytics_data.netdata_config_exporting_enabled, appconfig_get_boolean(&exporting_config, CONFIG_SECTION_EXPORTING, "enabled", CONFIG_BOOLEAN_NO) ? "true" : "false");`

Reply:
@vlvkobal please halp

Reply:
Yes, it is disabled by default.

Reply:
Ok, thanks both guys, pushed: https://github.com/netdata/netdata/pull/11100/commits/a078fe63a226ec847cd95016541b09132f80918f